lib/jormungandr/test/integration/Cardano/Faucet.hs

```diff
@@ -1,1483 +0,0 @@
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE QuasiQuotes #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeApplications #-}
-
-module Cardano.Faucet
```

Comment:
Renamed to `Cardano.Wallet.Jormungandr.Faucet` and, move the hard-coded mnemonics to `Test.Integration.Faucet`